### PR TITLE
Dotnet (RK-5747)

### PR DESCRIPTION
**Pull request by BenTheKlein**
@MickaelAlliel Can you help making sure the regression test are executing this?

### CODE REVIEW

**Comment by MickaelAlliel** [comment]
@BenTheKlein It is being executed automatically if you add the Jenkinsfile and Dockerfile.
It failed building though
https://jenkins.rookout.com/job/Rookout/job/deployment-examples/job/dotnet/2/console
`19:18:30  The command '/bin/sh -c dotnet publish -c Release -o out' returned a non-zero code: 1`
Pretty sure `publish` is not the right target here

File: dotnet-sample/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1 AS build-env
+WORKDIR /dotnet-sample
+
+# Copy csproj and restore as distinct layers
+COPY *.csproj ./
+RUN dotnet restore
+
+# Copy everything else and build
+COPY . ./
+RUN dotnet publish -c Release -o out
```
Comment (MickaelAlliel) [inline dotnet-sample/Dockerfile:10]: this command fails in jenkins

**Comment by BenTheKlein** [comment]
@MickaelAlliel how can we test that it actually works without merging it?
If we can't - please approve and ill make sure to publish a fix if needed

**Comment by MickaelAlliel** [comment]
@BenTheKlein important part is that the docker actually builds
https://jenkins.rookout.com/job/Rookout/job/deployment-examples/job/dotnet/3/
so thats good, just after merging see that the test actually pass, if it doesnt lets make a fix pr :)

**Comment by BenTheKlein** [comment]
@MickaelAlliel lets merge

**Pull request merged**